src/Microsoft.DotNet.Interactive.Tests/Server/SerializationTests.cs

```diff
@@ -70,7 +71,8 @@ public void All_event_types_are_round_trip_serializable(KernelEvent @event)
             var ignoredProperties = new HashSet<string>
             {
                 $"{nameof(CommandFailed)}.{nameof(CommandFailed.Exception)}",
-                $"{nameof(DisplayEvent)}.{nameof(DisplayEvent.Value)}"
+                $"{nameof(DisplayEvent)}.{nameof(DisplayEvent.Value)}",
```

Comment:
Odd that `DisplayEvent` is here, given it's abstract.

Reply:
is for the assertion, this is to filter out things that we know will be ignored at serialization time